src/Raven.Server/Documents/Queries/AST/JavascriptCodeQueryVisitor.cs

```diff
@@ -95,9 +100,72 @@ public override void VisitCompoundWhereExpression(BinaryExpression @where)
 
         public override void VisitMethod(MethodExpression expr)
         {
+            // The regex usage here is just temporary, to get things working. Ideally, we want to implement the string functions in Jint
+            if (expr.Name.Value.Equals("startswith", StringComparison.OrdinalIgnoreCase) && expr.Arguments.Count == 2)
+            {
+                _sb.Append("startsWith(");
+                VisitExpression(expr.Arguments[0]);
```

Comment:
Don't you need to escape the variable here, what if we have a quote in it?

Reply:
Not mean what you mean, escaping treament is in VisitValue